src/aiovodafone/api.py

```diff
@@ -111,6 +121,30 @@ async def get_device_type(
             except ClientConnectorSSLError:
                 _LOGGER.debug("Unable to login using protocol %s", protocol)
                 continue
+            except ClientConnectorError:
+                _LOGGER.debug("Unable to login using protocol %s", protocol)
+                continue
```

Comment:
You can add the ClientConnectorError to the previous except to avoid duplicating